src/ScottPlot/Plottable/Crosshair.cs

```diff
@@ -43,6 +43,11 @@ public class Crosshair : IPlottable
         /// </summary>
         public bool IsDateTimeX = false;
 
+        /// <summary>
+        /// If defined, this function will be used to generate labels from X position, value of StringFormatX and IsDateTimeX will be ignored
+        /// </summary>
+        public Func<double, string> ManualStringFormatterX = null;
```

Comment:
I think `TickLabelFormat` is ignored if it's a datetime axis, so this introduces an inconsistency.

I think it's probably better that it isn't ignored for datetime axes, but that change would mean changing the axis code as well.

Reply:
> I think `TickLabelFormat` is ignored if it's a datetime axis

Maybe there should be a `NumericFormatter` and `DateTimeFormatter`?

Reply:
I think it's fine for the user to figure that out themselves, as they should know whether or not it's a DateTime axis.

Although I suppose we could make it easier for them by having `DateTimeFormatter` take in a `DateTime` instead of a `double`.